src/main/java/com/rchicken/checkmoi2/global/HealthApi.java

```diff
@@ -16,8 +19,8 @@ public class HealthApi {
 
     @Operation(summary = "healthy", description = "어플리케이션의 실행 상태를 확인하는 API")
     @GetMapping
-    public String healthy() {
-        return "CheckMoi v2";
+    public ResponseEntity<SuccessResponse<String>> healthy() {
+        return ResponseEntity.ok(new SuccessResponse<>("CheckMoi v2", LocalDateTime.now()));
```

Comment:
```suggestion
        return ResponseEntity.ok(SuccessResponse.of("CheckMoi v2");
```
LocalDateTime을 이곳에서 관리하기보다는 관련 생성자를 하나 만들어서 관리하는게 어떨까요?

Reply:
맞네요 수빈님께서 제안해주신 방안이 더 좋아보입니다. 해당 방식으로 수정해서 올리겠습니다..!